main.js

```diff
@@ -103,70 +104,87 @@ function firstInstallTour(win) {
     progressButton.setAttribute('label', strings.progressButtonIntro);
     dismissLabel.textContent = strings.dismissLabel;
 
-    let xpos;
-    let outerRect = {};
-    let starttime;
-
-    let movePanel = function (timestamp, panel, dist, duration) {
-      let runtime = timestamp - starttime;
-      let progress = runtime / duration;
-      progress = Math.min(progress, 1);
-      panel.moveTo(xpos + (dist * progress), outerRect.y);
-      if (runtime < duration){
-        win.requestAnimationFrame(function (timestamp) {
-          movePanel(timestamp, panel, dist, duration);
-        });
+    if (win.reminderTour && get('extensions.tabcentertest1@mozilla.com.tourComplete') === 'true') {
+      progressButton.setAttribute('label', strings.progressButtonGo);
+      dismissLabel.textContent = strings.dismissLabelReminderTour;
+      progressButton.onclick = (e) => {
+        if (e.which !== 1) {
```

Comment:
I love that we're doing this by default now, instead of waiting for people to complain about right-clicks advancing the tour.  😀